src/stories/Blocks/header/header.scss

```diff
@@ -4,6 +4,11 @@
   display: grid;
   grid-template-columns: 1fr;
   background-color: $c-global-primary;
+  position: sticky;
+  transition-property: all;
+  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  transition-duration: 500ms;
+  z-index: 5000;
```

Comment:
The number 5000 seems like a somewhat arbitrary number compared to the current `z-index` values that range between -2 and 5. In my experience z-index quickly becomes a race to the bottom/top. Perhaps there is a meaningful way to manage this? Consider talking to colleagues or doing a bit of desk research.

Reply:
Very good point. I have introduced z-index variables now.